web/src/components/graph/timeaxis.js

```diff
@@ -20,15 +36,24 @@ const getTicksValuesFromTimeScale = (scale, count) => {
   const startTime = scale.domain()[0].valueOf();
   const endTime = scale.domain()[1].valueOf();
 
-  const precision = moment.duration(15, 'minutes').valueOf();
+  const precision = moment.duration(60, 'minutes').valueOf();
```

Comment:
This change should not affect anything, as we only have hourly values now

Reply:
FYI: This has changed in #4070